tools/ansible/tasks/wsu/main.yaml

```diff
@@ -99,3 +99,30 @@
       delegate_to: localhost
       shell: "oc adm certificate approve {{ item }}"
       with_items: "{{ node_csrs.stdout_lines }}"
+
+    # Get the bootstrapped windows node name. We're using the IP address of the Windows VM created
+    # in the inventory file and if the node has multiple internal IP's or external IP's it may not work well.
+    # TODO: Move this to a more deterministic way of identifying the node we just bootstrapped
+    - name: Get bootstrapped node name
+      delegate_to: localhost
+      shell: "oc get no -o wide |awk '/{{ inventory_hostname }}/ {print $1}'"
+      register: node_name
+      until: node_name.stdout != ""
+      retries: 3
+      delay: 5
+
+    # Check if worker label is already applied to the Windows nodes bootstrapped via WMCB
```

Comment:
Is there any harm in reapplying the label?

Reply:
The `oc label` command fails, if the label is already applied

Reply:
Reapplying of label fails unless the command overrides.